app/helpers/tabs_helper.rb

```diff
@@ -25,8 +25,9 @@ def tabs
   def tabs_for(user, resource)
     tabs_to_remove = []
     tabs_to_remove << "admin" unless user.has_editor_permissions?(resource)
+    tabs_to_remove << "admin" if resource.retired_format?
     tabs_to_remove << "unpublish" unless user.govuk_editor?
 
-    tabs.reject { |tab| tabs_to_remove.include?(tab.name) }
+    tabs.reject { |tab| tabs_to_remove.uniq.include?(tab.name) }
```

Comment:
You probably don't need the uniq here, include? doesn't care if a thing appears twice in a list. But no harm.

Reply:
You're correct, after some brief thought, I think I'm going to keep the `uniq` call so the array isn't populated with duplicates which may be surprising